types/k6/experimental/browser.d.ts

```diff
@@ -765,6 +765,53 @@ export interface BrowserContext {
     ): Page | null;
 }
 
+/**
+ * {@link ConsoleMessage} objects are dispatched by page via the
+ * `page.on('console')` event. For each console messages logged in the page
+ * there will be corresponding event in the k6 browser context.
+ *
+ * ```js
+ * // Listen for all console logs
+ * page.on('console', msg => console.log(msg.text()));
```

Comment:
Is this listening on all console logs and then printing them again but as k6 logs? Maybe the comment could be expanded a tiny bit to make that clear, otherwise to a new user maybe it could look like an infinite loop of logging? 😅 

Maybe something like "Listen for all console logs, and log them again to k6 logs" -- not 100% happy with that wording myself though.

Reply:
What about something like:
```suggestion
 * // Listen for all console logs in the page context and print them in the k6
 * // script context
 * page.on('console', msg => console.log(msg.text()));
```
And then for the following example [here](https://github.com/grafana/k6-DefinitelyTyped/blob/15025aade950fd71ade752247fea65488f775c34/types/k6/experimental/browser.d.ts#L777):
```
 * // Listen for all console events in the page context and print the ones that
 * // are errors
```
Is quite confusing to clearly reference what happens in the browser instance and what happens in k6 browser as we use same namings/elements...

Reply:
What about?

"Listen for console log messages on the browser page and output them in the test log."

Reply:
> What about?
> "Listen for console log messages on the browser page and output them in the test logs."

This works for me. Should it be `on the browser page` or `in the browser page`? Thoughts, @ankur22 ?

Reply:
> This works for me. Should it be `on the browser page` or `in the browser page`? Thoughts, @ankur22 ?

I think it should be `in the browser page` 👍

Reply:
Fixed in [a99c3cf](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66549/commits/a99c3cfa84a548103a8aaaa78b10ec043130dc09).